### PR TITLE
Impute location labels

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: src/auto_code_surveys.py [hunk 1]
```diff
@@ -31,16 +31,43 @@ def auto_code_surveys(cls, user, data, phone_uuid_table, coda_output_dir):
             if plan.cleaner is not None:
                 CleaningUtils.apply_cleaner_to_traced_data_iterable(user, data, plan.raw_field, plan.coded_field,
                                                                     plan.cleaner, plan.code_scheme)
+
+        # For any locations where the cleaners assigned a code to a constituency, set the county code to NC
```
Comment (as2388) [inline src/auto_code_surveys.py]: Given that we're not applying cleaners to the locations, is this block needed?

File: src/auto_code_surveys.py [hunk 2]
```diff
@@ -31,16 +31,43 @@ def auto_code_surveys(cls, user, data, phone_uuid_table, coda_output_dir):
...
+            if plan.raw_field == "location_raw":
+                continue
             TracedDataCodaV2IO.compute_message_ids(user, data, plan.raw_field, plan.id_field)
             coda_output_path = path.join(coda_output_dir, plan.coda_filename)
             with open(coda_output_path, 'w') as f:
                 TracedDataCodaV2IO.export_traced_data_iterable_to_coda_2(
                     data, plan.raw_field, plan.time_field, plan.id_field, {plan.coded_field: plan.code_scheme}, f
                 )
+
+        # Output location scheme to coda for manual verification + coding
```
Comment (as2388) [inline src/auto_code_surveys.py]: same here (about this being needed)

File: src/lib/pipeline_configuration.py [hunk 3]
```diff
@@ -207,9 +208,30 @@ class PipelineConfiguration(object):
...
                     analysis_file_key="organizations_",
                     cleaner=None,
-                    code_scheme=CodeSchemes.ORGANIZATIONS)
+                    code_scheme=CodeSchemes.ORGANIZATIONS),
     ]
-    
+
+    LOCATION_CODING_PLANS = [
+
+        CodingPlan(raw_field="constituency_raw",
```
Comment (as2388) [inline src/lib/pipeline_configuration.py:216]: In hindsight this is arguably better called "location_raw".

Comment (as2388) [reply src/lib/pipeline_configuration.py:216]: Don't have to change that in this PR if it would cause too much chaos now

**Review by as2388 — CHANGES_REQUESTED** [review]
Thanks @IsaackMwenda, and with perfect timing for when I needed this on AHADI :)

Requesting changes because of my questions in auto_code_surveys.

File: src/auto_code_surveys.py (hunk 1 above)
Comment (IsaackMwenda) [reply src/auto_code_surveys.py]: No was just getting ready for tests for other Kenyan based project incase we get time to build the cleaners. Happy to revert to previous flow and submit this as a pr later on.

File: src/lib/pipeline_configuration.py (hunk 3 above)
Comment (IsaackMwenda) [reply src/lib/pipeline_configuration.py:216]: I agree too location_raw is more descriptive. However due to the project current timelines I opted to leave it as is for now.

**Comment by IsaackMwenda** [comment]
refactored auto-code surveys PTAL @as2388

File: src/apply_manual_codes.py [hunk 4]
```diff
@@ -117,9 +174,11 @@ def apply_manual_codes(cls, user, data, coda_input_dir):
         # TODO: Investigate/address the cause of this.
         sys.setrecursionlimit(10000)
         for plan in PipelineConfiguration.DEMOGS_CODING_PLANS:
+            print(plan.raw_field)
```
Comment (as2388) [inline src/apply_manual_codes.py]: Remove or use log

**Review by as2388 — APPROVED** [review]

**Pull request merged**